stan/math/rev/fun/gamma_p.hpp

```diff
@@ -78,11 +78,11 @@ class gamma_p_dv_vari : public op_dv_vari {
       : op_dv_vari(gamma_p(a, bvi->val_), a, bvi) {}
   void chain() {
     if (is_inf(ad_)) {
-      bvi_->adj_ += NOT_A_NUMBER;
+      bvi_->adj_ = NOT_A_NUMBER;
       return;
     }
     if (is_inf(bvi_->val_)) {
-      bvi_->adj_ += NOT_A_NUMBER;
+      bvi_->adj_ = NOT_A_NUMBER;
```

Comment:
[optional] It looks like the conversion on gamma_p never got finished.

Reply:
I just did a ctrl+f for `+= NOT_A_NUMBER` and that's why these changed. I'm going to do the vv ones in a future PR